src/redux/reducers/searchText.js

```diff
@@ -0,0 +1,198 @@
+import { handleActions } from 'redux-actions';
+import actionTypes from '../actionTypes';
+import { searchMessages } from '../../utils/searchMessages';
+
+const defaultState = {
+  search: false,
```

Comment:
What does `search` signify? Can you give it a more apt name like `isSearchBarActive` depending on whatever it means.